DDPatchForSR/Package.swift

```diff
@@ -0,0 +1,62 @@
+// swift-tools-version: 5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "DatadogSDK",
+    platforms: [
+        .iOS(.v11),
+        .tvOS(.v11)
+    ],
+    products: [
+        .library(
+            name: "Datadog",
+            targets: ["Datadog"]
+        ),
+        .library(
+            name: "DatadogObjc",
+            targets: ["DatadogObjc"]
+        ),
+        .library(
+            name: "DatadogCrashReporting",
+            targets: ["DatadogCrashReporting"]
+        ),
+    ],
+    dependencies: [
+        .package(name: "PLCrashReporter", url: "https://github.com/microsoft/plcrashreporter.git", from: "1.11.0"),
+    ],
+    targets: [
+        .target(
+            name: "Datadog",
+            dependencies: [
+                "_Datadog_Private",
+            ],
+            swiftSettings: [
+                .define("SPM_BUILD"),
+                .define("DD_SDK_ENABLE_EXPERIMENTAL_APIS"),
+            ]
```

Comment:
It is not possible to add `path: ..` as SPM will complain that sources cannot be located in outside of package directory. Symbolic links to `Sources` and `Tests` workaround this limit.